tools/tcoffee/tcoffee.xml

```diff
@@ -0,0 +1,186 @@
+            <option value="seq_pair">seq</option>
+            <option value="externprofile_pair">externprofile</option>
+            <option value="hh_pair">hh</option>
+            <option value="profile_pair">profile</option>
+            <option value="cdna_fast_pair">cdna fast</option>
+            <option value="cdna_cfast_pair">cdna cfast</option>
+            <option value="clustalw_pair">ClustalW</option>
+            <option value="mafft_pair">mafft</option>
+            <option value="mafftjtt_pair">mafftjtt</option>
+            <option value="mafftgins_pair">mafftgins</option>
+            <option value="dialignt_pair">dialign-t</option>
+            <option value="poa_pair">poa</option>
+            <option value="probcons_pair">ProbCons</option>
+            <option value="muscle_pair">Muscle</option>
+            <option value="t_coffee_pair">T-Coffee</option>
+            <option value="pcma_pair">pcma</option>
+            <option value="kalign_pair">kalign</option>
+            <option value="amap_pair">amap</option>
+            <option value="proda_pair">proda</option>
+            <option value="prank_pair">prank</option>
+            <option value="consan_pair">consan</option>
+        </param>
+
+        <param name="method2" type="select" label="Pairwise Structural Alignment Method">
+            <option value="">Default</option>
+            <option value="align_pdbpair">align_pdb</option>
+            <option value="lalign_pdbpair">lalign_pdb</option>
+            <option value="extern_pdbpair">extern_pdb</option>
+            <option value="thread_pair">thread</option>
+            <option value="fugue_pair">fugue</option>
+            <option value="pdb_pair">pdb</option>
+            <option value="sap_pair">sap</option>
+            <option value="mustang_pair">mustang</option>
+            <option value="tmalign_pair">tmalign</option>
+        </param>
+
+        <param name="method3" type="select" label="Multiple Sequence Alignment Method">
+            <option value="">Default</option>
+            <option value="clustalw_msa">ClustalW</option>
+            <option value="mafft_msa">mafft</option>
+            <option value="mafftjtt_msa">mafftjtt</option>
+            <option value="mafftgins_msa">mafftgins</option>
+            <option value="dialigntx_msa">dialign-tx</option>
+            <option value="dialignt_msa">dialign-t</option>
+            <option value="poa_msa">poa</option>
+            <option value="probcons_msa">ProbCons</option>
+            <option value="muscle_msa">Muscle</option>
+            <option value="t_coffee_msa">T-Coffee</option>
+            <option value="pcma_msa">pcma</option>
+            <option value="kalign_msa">kalign</option>
+            <option value="amap_msa">amap</option>
+            <option value="proda_msa">proda</option>
+            <option value="prank_msa">prank</option>
+        </param>
+
+        <section name="out" title="Output Format" expanded="True">
```

Comment:
You can use here a `type="select" multiple="true"` like here: https://github.com/bgruening/galaxytools/blob/master/tools/antismash/antismash.xml#L180